src/components/atoms/Button/index.js

```diff
@@ -2,14 +2,11 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import './style.scss';
 
-const Button = (props) => {
-  const { text, green } = props;
-  return (
-    <button type="button" className={`App__Button ${green ? 'green' : 'white'} `}>
-      {text}
-    </button>
-  );
-};
+const Button = ({ text, green }) => (
```

Comment:
Lets try and re-write this like,
```suggestion
const Button = ({ children, color='white', className='', ...props }) => (
  <button {...props} type="button" className={`App__Button ${color} ${className}`}>
    {children}
   </button>
  )
  
 Button.propTypes = {
   color: PropTypes.oneOf(['green', 'white'])
 }
```